conf/uppmax.config

```diff
@@ -13,8 +16,36 @@ process {
   memory = '16 GB'
   time = '48h'
   clusterOptions = {
-    '-A <PROJECT>'
+    "-A $params.project"
   }
+
+  errorStrategy = task.exitStatus == 143 ? 'retry' : 'ignore'
+  maxRetries = 3
+  maxErrors = '-1'
+
+  // Module specific retrying
+  $fastqc.errorStrategy = task.exitStatus == 143 ? 'retry' : 'warning'
```

Comment:
There are a bunch of errors here that were mine in the RNA pipeline, that I now know about. I'll replace this when I do the new refactoring again shortly though.

For reference, this should be `$fastqc.errorStrategy = { task.exitStatus == 143 ? 'retry' : 'warning' }`. It has to be inside `{}` if you want it to be evaluated at run time. You need this, as `task.exitStatus` doesn't exist until the process has run.